tests/test_auto_and_edge_cases.py

```diff
@@ -23,3 +24,11 @@ def test_mixed_language_context():
     text = "The result is ٤٥٦ and also ۰۱۲"
     expected = "The result is 456 and also 012"
     assert convert(text, target=NumeralSystem.ENGLISH) == expected
+
+
+def test_detect_system_default():
+    assert detect_system(" ") == NumeralSystem.ENGLISH
```

Comment:
We should mention this corner case behaviour in the readme; Something like "in case which no numeral digits appear in the text, we assume the default nominal system to be English"